### PR TITLE
docs: refine flux inspired docs to make pattern inspiration clearer

### PR DESCRIPTION
**Pull request by chrisk-7777**
## Related Issues or Discussions

Addresses discussion 1863

## Summary

Try to highlight the similarities, but also differences, from standard flux practices. 

## Check List

- [x] `yarn run prettier` for formatting code and docs

**Note** I had to remove `<>` from the codesandbox link as it was causing issues when running the docs locally, see screenshot. 
CleanShot 2023-06-17 at 21 13 08

### CODE REVIEW

**Comment by chrisk-7777** [comment]
Before merging, I'd like to get feedback in the original discussion #1863

**Comment by chrisk-7777** [comment]
This is ready to merge / approve now as per comment in discussion https://github.com/pmndrs/zustand/discussions/1863#discussioncomment-6203882

File: docs/guides/flux-inspired-practice.md [hunk 1]
```diff
@@ -3,27 +3,47 @@ title: Flux inspired practice
...
+### Single store
+
+Your applications global state should be located in a single Zustand store.
+
+If you have a large application, Zustand supports [splitting the store into slices](./slices-pattern).
+
+### Use `set` / `setState` to update the store
+
+Always use `set` (or `setState`) to perform updates to your store.
+`set` (and `setState`) ensures the described update is immutably performed to the store.
```
Comment (dai-shi) [inline docs/guides/flux-inspired-practice.md:25]: Immutability isn't guaranteed by `set/setState`. Developers has to guarantee it.

**Review by dai-shi — APPROVED** [review]
Thanks for the fix and improvement!!

File: docs/guides/flux-inspired-practice.md (hunk 1 above)
Comment (chrisk-7777) [reply docs/guides/flux-inspired-practice.md]: Ah yes, good point! I have updated in [additional commit](https://github.com/pmndrs/zustand/pull/1874/commits/f252b54dd7143172132c9518cb798547b324e038)

Comment (dai-shi) [reply docs/guides/flux-inspired-practice.md]: thanks!

**Pull request merged**